docs/recipes/nvhpc-a100-with-cuda-and-mpi-example.md

```diff
@@ -0,0 +1,262 @@
+---
+tags:
+ - Engaging
+ - Howto Recipes
+ - nvhpc
+ - mpi
+ - cuda
+ - cuda aware mpi
+---
+
+# Example of a minimal program using the nvhpc stack with CUDA aware MPI
+
+## About NVHPC
+
+NVHPC is an integrated collection of software tools and libraries distributed by NVidia. An overview document describing nvhpc 
+can be found [here](https://developer.nvidia.com/hpc-sdk).
```

Comment:
Do we want external links to open in a new tab? I was able to figure out how to add this for the SuperCloud site so I could add it here as well.

Reply:
@lmilechin - defintiely. I tried https://github.com/mkdocs/mkdocs/issues/1958 , but failed. What did you use on SC.

Reply:
Here is the PR: https://github.com/mit-supercloud/supercloud-docs-edit/pull/8. I can put this into orcd-docs in a separate PR.

Reply:
That one! I tried the other one. 

I'll add it.

Chris